frontend/src/pages/add/BookAdd.tsx

```diff
@@ -98,13 +98,11 @@ const COLUMNS: TableColumn[] = [
   },
 ];
 
-class BookAdd extends React.Component<{}, BookAddState> {
-  constructor(props = {}) {
-    super(props);
-    this.state = { inputBoxText: "", data: DATA };
-  }
+export default function BookAdd() {
+  const [textBox, setTextBox] = useState("");
+  const [books, setBooks] = useState(DATA);
```

Comment:
Why is there a book state. Is it code that I wrote which no longer will be needed for the new react strucuture? Can you remove it for me, if yes. Thanks.